src/main/java/com/plaid/client/response/AssetReportGetResponse.java

```diff
@@ -0,0 +1,181 @@
+    private String accountId;
+    private String mask;
+    private String name;
+    private String officialName;
+    private String type;
+    private String subtype;
+    private List<Owner> owners;
+    private Balance balances;
+    private List<HistoricalBalance> historicalBalances;
+    private List<Transaction> transactions;
+    private int daysAvailable;
+
+    public String getAccountId() { return accountId; }
+    public String getMask() { return mask; }
+    public String getName() { return name; }
+    public String getOfficialName() { return officialName; }
+    public String getType() { return type; }
+    public String getSubtype() { return subtype; }
+    public List<Owner> getOwners() { return owners; }
+    public Balance getBalances() { return balances; }
+    public List<HistoricalBalance> getHistoricalBalances() { return historicalBalances; }
+    public List<Transaction> getTransactions() { return transactions; }
+    public int getDaysAvailable() { return daysAvailable; }
+  }
+
+  public final static class Owner {
+    private List<String> names;
+    private List<PhoneNumber> phoneNumbers;
+    private List<Email> emails;
+    private List<Address> addresses;
+
+    public List<String> getNames() { return names; }
+    public List<PhoneNumber> getPhoneNumbers() { return phoneNumbers; }
+    public List<Email> getEmails() { return emails; }
+    public List<Address> getAddresses() { return addresses; }
+  }
+
+  public final static class PhoneNumber {
+    private String data;
+    private boolean primary;
+    private String type;
+
+    public String getData() { return data; }
+    public boolean isPrimary() { return primary; }
+    public String getType() { return type; }
+  }
+
+  public final static class Email {
+    private String data;
+    private boolean primary;
+    private String type;
+
+    public String getData() { return data; }
+    public boolean isPrimary() { return primary; }
+    public String getType() { return type; }
+  }
+
+  public final static class Address {
+    private AddressData data;
+    private boolean primary;
+
+    public AddressData getData() { return data; }
+    public boolean isPrimary() { return primary; }
+  }
+
+  public final static class AddressData {
+    private String city;
+    private String state;
+    private String street;
+    private String zip;
+
+    public String getCity() { return city; }
+    public String getState() { return state; }
+    public String getStreet() { return street; }
+    public String getZip() { return zip; }
+  }
+
+  public final static class Balance {
+    private Double avaliable;
+    private Double current;
+
+    public Double getAvaliable() { return avaliable; }
+    public Double getCurrent() { return current; }
+  }
+
+  public final static class HistoricalBalance {
+    private Date date;
+    private Double current;
+
+    public Date getDate() { return date; }
+    public Double getCurrent() { return current; }
```

Comment:
is there a reason this is nullable?

Reply:
I wasn't sure how GSON would handle a missing field, but yeah there's no reason we won't get those fields back

Reply:
I just kind of want us to be consistent. for example the daysAvailable in the other response is an int instead of an Integer. We don't do a good job of using @Nullable, so it's unclear as to if things are Double because everything uses the boxed type (which we don't) or if it's Double because it's nullable.

for safety i'd rather have all the numeric types be Boxed

Reply:
Yeah that makes sense- I'll make it the unboxed double to be consistent

Reply:
Although, we do have a bunch of the boxed Doubles in the code base:
```
src/main/java/com/plaid/client/response/Account.java:    private Double available;
src/main/java/com/plaid/client/response/Account.java:    private Double current;
src/main/java/com/plaid/client/response/Account.java:    private Double limit;
src/main/java/com/plaid/client/response/Account.java:    public Double getAvailable() {
src/main/java/com/plaid/client/response/Account.java:    public Double getCurrent() {
src/main/java/com/plaid/client/response/Account.java:    public Double getLimit() {
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    private Double lastPaymentAmount;
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    private Double lastStatementBalance;
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    private Double minimumPaymentAmount;
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    public Double getLastPaymentAmount() {
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    public Double getLastStatementBalance() {
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    public Double getMinimumPaymentAmount() {
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    private Double apr;
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    private Double balanceSubjectToApr;
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    private Double interestChargeAmount;
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    public Double getApr() {
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    public Double getBalanceSubjectToApr() {
src/main/java/com/plaid/client/response/CreditDetailsGetResponse.java:    public Double getInterestChargeAmount() {
src/main/java/com/plaid/client/response/TransactionsGetResponse.java:    private Double amount;
src/main/java/com/plaid/client/response/TransactionsGetResponse.java:    public Double getAmount() {
src/main/java/com/plaid/client/response/TransactionsGetResponse.java:      private Double lat;
src/main/java/com/plaid/client/response/TransactionsGetResponse.java:      private Double lon;
src/main/java/com/plaid/client/response/TransactionsGetResponse.java:      public Double getLat() {
src/main/java/com/plaid/client/response/TransactionsGetResponse.java:      public Double getLon() {
src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java:                                      Double expectedBalanceAvailable,
src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java:                                      Double expectedBalanceCurrent,
src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java:                                      Double expectedBalanceLimit,
```

Reply:
if you're even a bit worried, it's fine to keep it as boxed. so long as all the assets stuff is either boxed or unboxed for consistency